src/multiclient/client_wrapper.h

```diff
@@ -21,20 +25,29 @@ struct ClientConfig {
 
 class ClientWrapper : public td::actor::Actor {
 public:
-  explicit ClientWrapper(ClientConfig config);
+  explicit ClientWrapper(ClientConfig config, std::shared_ptr<tonlib::TonlibCallback> callback);
 
   void start_up() override;
   void alarm() override;
 
   template <typename T>
   void send_request(T&& req, td::Promise<typename T::ReturnType> promise);
+  void send_callback_request(uint64_t request_id, ton::tonlib_api::object_ptr<ton::tonlib_api::Function>&& request);
+  void send_request_json(uint64_t request_id, std::string req, td::Promise<std::string> promise);
+
+  void on_cb_result(uint64_t id, tonlib_api::object_ptr<tonlib_api::Object> result);
+  void on_cb_error(uint64_t id, tonlib_api::object_ptr<tonlib_api::error> error);
```

Comment:
nit: these 2 can be made private if you define `ClientWrapperCallback` inside `ClientWrapper::start_up`